front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorButtons.tsx

```diff
@@ -112,9 +113,13 @@ function RollingStockEditorButtons({
 
   return (
     <div
-      className={`rollingstock-editor-buttons ${
-        isCondensed ? 'condensed flex-column align-items-center rounded-right' : ''
-      } d-flex p-1`}
+      className={cx(
+        'rollingstock-editor-buttons',
+        {
+          'condensed flex-column align-items-center rounded-right': isCondensed,
+        },
+        'd-flex p-1'
+      )}
```

Comment:
Same here, should we put the boolean classes at the end ? It's harder to read in this example by putting it in the middle

Reply:
Done